tests/integration/base_file_test.go

```diff
@@ -43,11 +43,23 @@ func runFileE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.S
 	createFilesystemConsumerPod(helper, k8sh, s, namespace, filesystemName)
 	err := writeAndReadToFilesystem(helper, k8sh, s, namespace, filePodName, "test_file")
 	require.Nil(s.T(), err)
+
+	createNFSDaemons(helper, k8sh, s, namespace, filesystemName)
+
 	downscaleMetadataServers(helper, k8sh, s, namespace, filesystemName)
 	cleanupFilesystemConsumer(helper, k8sh, s, namespace, filesystemName, filePodName)
 	cleanupFilesystem(helper, k8sh, s, namespace, filesystemName)
 }
 
+func createNFSDaemons(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.Suite, namespace string, filesystemName string) {
+	name := "my-nfs"
+	err := helper.NFSClient.Create(namespace, name, filesystemName+"-data0", 2)
+	require.Nil(s.T(), err)
+
+	err = helper.NFSClient.Delete(namespace, name)
```

Comment:
Function is called `createNFSDaemons`, but it creates and deletes the NFS Daemon?

Reply:
I've renamed it to `testNFSDaemons()`. It's a very simple test, we should make it smarter soon though.